From 235ad72dd9efb9886b22a3b9495067c889d6cf74 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Mon, 20 Jul 2020 02:52:19 +0000
Subject: [PATCH 1/2] gpu: imx: imx-hdp: drop unused function to fix build
 warning

The static function imx_hdp_link_rate is not used in hdp driver,
so drop it to fix the compile warning.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index aff5397d1957..12b55f937b8b 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -682,20 +682,6 @@ void imx8qm_ipg_clock_set_rate(struct hdp_clks *clks)
 	}
 }
 
-static u8 imx_hdp_link_rate(struct drm_display_mode *mode)
-{
-#ifndef CONFIG_ARCH_LAYERSCAPE
-	if (mode->clock < 297000)
-#else
-	if (mode->clock < 74250)
-#endif
-		return AFE_LINK_RATE_1_6;
-	else if (mode->clock > 297000)
-		return AFE_LINK_RATE_5_4;
-	else
-		return AFE_LINK_RATE_2_7;
-}
-
 static void imx_hdp_mode_setup(struct imx_hdp *hdp,
 			       const struct drm_display_mode *mode)
 {
-- 
2.17.1

